src/CancelRequestDialog.test.js

```diff
@@ -0,0 +1,113 @@
+import {
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../test/jest/__mock__';
+
+import CancelRequestDialog from './CancelRequestDialog';
+
+const labelIds = {
+  modalLabel: 'ui-requests.cancel.modalLabel',
+  modalText: 'ui-requests.cancel.requestWillBeCancelled',
+  cancelRequestDialogCancel: 'stripes-core.button.confirm',
+  cancelRequestDialogClose: 'stripes-core.button.back',
+};
+const testIds = {
+  cancelRequestDialog: 'cancelRequestDialog',
+  cancelRequestDialogCancel: 'cancelRequestDialogCancel',
+  cancelRequestDialogClose: 'cancelRequestDialogClose',
+};
+
+describe('CancelRequestDialog', () => {
```

Comment:
add test to assert
1. if Select component is rendered.
2. if 'Confirm' button is disabled after clicking it and the modal is closed after a while?
3. if TextArea component with label 'Additional information for patron' is rendered?

Reply:
Fixed 1 and 3